market-generator/src/main/java/com/market/banica/generator/state/MarketStateObserver.java

```diff
@@ -0,0 +1,34 @@
+package com.market.banica.generator.state;
+
+import com.market.banica.generator.service.MarketStateImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.EnableMBeanExport;
+import org.springframework.jmx.export.annotation.ManagedOperation;
+import org.springframework.jmx.export.annotation.ManagedResource;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@EnableMBeanExport
+@ManagedResource
+@Service
+public class MarketStateObserver {
```

Comment:
Does the orderbook jmx observer show the aggregated products from all markets or all products per market ?
If it shows them aggregated, currently there is no way to see what product we have in a specific market and this observer is needed otherwise we don't need it.